src/routes/(console)/organization-[organization]/billing/+page.ts

```diff
@@ -24,17 +25,34 @@ export const load: PageLoad = async ({ parent, depends }) => {
               .catch(() => null)
         : null;
 
-    const [paymentMethods, addressList, aggregationList, billingAddress] = await Promise.all([
+    const [
+        paymentMethods,
+        addressList,
+        aggregationList,
+        billingAddress,
+        currentPlan,
+        creditList,
+        invoices
+    ] = await Promise.all([
         sdk.forConsole.billing.listPaymentMethods(),
         sdk.forConsole.billing.listAddresses(),
         sdk.forConsole.billing.listAggregation(organization.$id),
-        billingAddressPromise
+        billingAddressPromise,
+        sdk.forConsole.billing.getPlan(organization.$id),
+        sdk.forConsole.billing.listCredits(organization.$id, [Query.offset(0)]),
```

Comment:
```suggestion
        sdk.forConsole.billing.listCredits(organization.$id),
```
pretty sure we can skip the offset query of `0`

Reply:
changed